bin/compile

```diff
@@ -55,11 +55,11 @@ for DEB in $(ls -1 $APT_CACHE_DIR/archives/*.deb); do
 done
 
 echo -n "-----> Installing ngrok... "
-curl --silent -o ngrok.zip -L "https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip" | indent
+curl --silent -o ngrok.zip -L "https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-windows-amd64.zip" | indent
```

Comment:
we definitely don't want the windows version (Heroku is linux only). But I can merge this if we change this back.